README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
```

Comment:
What is the `brew` thing of which you speak?   (i.e. can you put this in a Max OS X specific section/note).